collectors/python.d.plugin/alarms/metadata.yaml

```diff
@@ -3,20 +3,25 @@ meta:
   module_name: alarms
   monitored_instance:
     name: python.d alarms
-    link: ''
-    categories: []
+    link: https://learn.netdata.cloud/docs/data-collection/netdata-specific/alarms
+    categories:
+      - data-collection.other
     icon_filename: ''
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
     description: ''
-  keywords: []
+  keywords:
+    - alarms
+    - netdata
   most_popular: false
 overview:
   data_collection:
-    metrics_description: ''
-    method_description: ''
+    metrics_description: |
+      This collector creates an 'Alarms' menu with one line plot of `alarms.status`.
+    method_description: |
+      Alarm status is read from the Netdata agent rest api [`/api/v1/alarms?all`](https://learn.netdata.cloud/api#/alerts/alerts1).
```

Comment:
same in this file, I think there is some autodetection going on, check what I attempted on this https://github.com/netdata/netdata/pull/15429/files#diff-0897faec3e60253ba9151128484620d31b39d41e012a2750e868eab4eae82235R36

Reply:
tried to fix in my commit